app/i18next.config.client.js

```diff
@@ -0,0 +1,26 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+import config from '../configs/app.config';
+
+const i18nextOptions = {
+  fallbackLng: config.fallbackLng,
+  interpolation: {
+    escapeValue: false,
+  },
+  lng: 'en',
+  react: {
+    wait: false,
+  },
+  saveMissing: true,
+  whitelist: config.languages,
```

Comment:
Double check this key. I think it may have been deprecated for `supportedLngs` -- I would check all the keys tbh.